sql-plugin/src/main/330until340/scala/org/apache/spark/rapids/ShimTrampolineUtil.scala

```diff
@@ -24,6 +24,6 @@ object ShimTrampolineUtil {
     // These are the arguments required by SparkDateTimeException class to create error message.
     val errorClass = "CAST_INVALID_INPUT"
     val messageParameters = Array("DOUBLE", "TIMESTAMP", SQLConf.ANSI_ENABLED.key)
-    new SparkDateTimeException(errorClass, Array(infOrNan) ++ messageParameters)
+    throw new SparkDateTimeException(errorClass, Array(infOrNan) ++ messageParameters)
```

Comment:
Nit: If this is going to throw, it'd be nice if the method stated as such.  For example, RapidsErrorUtils creates the exception classes and returns them, it does not throw them, so there's precedent for methods to not throw the exceptions they are creating.

Alternatively, we could update the calling code to do the throw, which is what this originally seems intended to do.

Reply:
Thanks @jlowe  for the suggestion. I have updated the calling code to do the throw. PTAL.